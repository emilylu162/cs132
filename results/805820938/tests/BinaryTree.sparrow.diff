18,19c18,26
< --- Error (Explicit) ---
< null pointer
---
> 4
> 8
> 14
> 16
> 20
> 24
> 28
> 0
> 0
