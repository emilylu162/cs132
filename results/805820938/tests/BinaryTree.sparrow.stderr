WARNING: A command line option has enabled the Security Manager
WARNING: The Security Manager is deprecated and will be removed in a future release
Error: Move_Id_Id with lhs v233 detected. This should not happen.
Error: Function argument v233_15 is not spilled or allocated to a register.
